core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -181,7 +181,7 @@ export class QuestionBackendApiService {
    * Returns a list of questions based on the list of skill ids and number
    * of questions requested.
    */
-  fetchQuestions(
+  async fetchQuestionsAsync(
```

Comment:
Do we need to rename the function here as well?

https://github.com/oppia/oppia/blob/eec98fce432d1cdb585fef8294dadece0c4ed993/core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts#L76

Reply:
I don't think so, coz renaming the function to `fetchQuestionsAsync` would throw a TypeError saying function doesn't exist. 
<img width="589" alt="Screenshot 2021-03-05 at 21 25 11" src="https://user-images.githubusercontent.com/19875783/110139645-47364600-7df9-11eb-99bf-979d43dd01a0.png">

And that is something I actually fixed in https://github.com/oppia/oppia/pull/12021/commits/0e12de4d1b30dfdd0024b8d867434e79a27adc93

Reply:
That looks like you need to modify the test as well. We seem to be mocking the `QuestionBackendApiService` there.

https://github.com/oppia/oppia/blob/3814a20931432f18faf88fc027840292d2b1ed57/core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.controller.spec.ts#L147-L155